README.md

```diff
@@ -9,7 +9,7 @@ LORIS (Longitudinal Online Research and Imaging System) is a self-hosted web app
 * Try the LORIS demo instance at https://demo.loris.ca.
 
 This Readme covers installation of the LORIS <b>v22.*</b> release on <b>Ubuntu</b>.
-([CentOS Readme also available](./README.CentOS7.md)).
+([CentOS Readme also available](:./docs/wiki/00%20-%20SERVER%20INSTALL%20AND%20CONFIGURATION/01%20-%20LORIS%20Install/CentOS/README.CentOS7.md)).
```

Comment:
This link doesn't seem to be rendering properly. I don't think you need to URL encode the spaces.